benefits/core/templates/core/base.html

```diff
@@ -68,6 +68,37 @@
 
     <main id="main-content" role="main">
       {% block main_content %}
+        <div class="container main-row">
+          <div class="row">
+            {% block nav-buttons %}
+            {% endblock nav-buttons %}
+          </div>
+          <div class="row justify-content-center">
+            <div id="headline" class="col-lg-10 text-sm-start text-lg-center">
```

Comment:
I would prefer to not have `text-sm-start text-lg-center` here. This is now defined in the CSS itself for `h1`. I am also not sure if we need `id=headline` - what would it be used for?

Reply:
I'll remove the `id="headline"`. I think it's leftover from a previous idea. I'm also fine with removing those `text-` classes if the CSS already has it.